requirements.txt

```diff
@@ -1,4 +1,4 @@
 scikit-multilearn
 arff
 python-dateutil<2.8.1,>=2.1
--e git+git://github.com/wellcometrust/WellcomeML.git@ca5d1c38f28a50aacdefa5b5b47647389194e826#egg=wellcomeml[deep-learning]
+-e git+git://github.com/wellcometrust/WellcomeML.git@14668e8650516363453a85bc2a5546cfe8811951#egg=wellcomeml[deep-learning]
```

Comment:
Do you not want to use the pypi version?

Reply:
haha, because I so often submit a feature and fix I keep pointing to the latest commit to develop new features and use the latest published when I "publish" a new version